apps/www/styles/index.css

```diff
@@ -205,7 +205,7 @@ h6:not(.overwrite) {
 }
 
 .sbui-tabs--alt div[role='tablist'] .sbui-btn-primary {
-  @apply dark:bg-scale-300 bg-gray-800 text-white dark:text-white;
+  @apply dark:bg-surface-100 bg-gray-800 text-white dark:text-white;
```

Comment:
From the `text-white dark:text-white` it looks like it should always have a dark bg. I'd leave it out for now in this PR since I haven't found an actual example to see the output.